dbfit-java/core/src/main/java/dbfit/util/DbParameterAccessor.java

```diff
@@ -40,17 +42,18 @@ public DbParameterAccessor clone() {
     }
 
     @SuppressWarnings("unchecked")
-    public DbParameterAccessor(String name, Direction direction, int sqlType, Class javaType, int position) {
-        this(name, direction, sqlType, null, javaType, position);
+    public DbParameterAccessor(String name, Direction direction, int sqlType, Class javaType, int position, Map<Class<?>, TypeTransformer> typeSpecifers) {
```

Comment:
I was thinking that I shouldn't probably be typing `Map<Class<?>, TypeTransformer>` so many times throughout the code.

How about something like this: -

```java
public class TypeTransformers {
    private Map<Class<?>, TypeTransformer> typeXforms;

    public TypeTransformers(Map<Class<?>, TypeTransformer> typeXformers) {
        this.typeXforms = typeXformers;
    }

    public TypeTransformer get(Class<?> cls) {
        return typeXforms.get(cls);
    }
}
```

So we should revist PR #443 with too then first?